src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts

```diff
@@ -602,6 +602,7 @@ export class FormTransferTransactionTs extends FormTransactionBase {
     @Watch('selectedSigner')
     onSelectedSignerChange() {
         this.formItems.signerAddress = this.selectedSigner.address.plain();
+        this.resetForm();
```

Comment:
Wouldn't this break this change? => https://github.com/symbol/desktop-wallet/pull/1642
Should we just update the necessary parts instead?

Reply:
nope, it wouldn't break it. After having this sticky account feature it works as expected.

Reply:
After sending from multisig "From" selector stays selected on previously choosen multisig.

Reply:
Let's say as a user, I filled the transfer form and then noticed that I've chosen the wrong account, currently, all the other fields remain the same when I changed the signer but with this change, I'll have to re-enter all the information since it'll reset the form, right? I'm not sure if that's the behaviour we want to have in the transfer form tbh, @cryptoBeliever @coiki any ideas?

Reply:
As we need the form to be reactive especially when it comes to the held mosaics in each account, I believe we need to reset fields on the change of signer.

Reply:
@yilmazbahadir you are right. I thought about other case here. 
@bassemmagdy would be possible to refresh only mosaics?

Reply:
we can only reset the held mosaics (dropdown and maybe the amount - unsure) rather than resetting the form, right? (If it won't break the reactivity ofc)

https://github.com/symbol/desktop-wallet/pull/1680#discussion_r699210817
> Should we just update the necessary parts instead?

Reply:
Form is cleaned after tx confirm. So we have regression with this fix -> https://github.com/symbol/desktop-wallet/issues/1577